### DISCUSSION TITLE
Weird "hour produced" readings?

### DISCUSSION THREAD

**Discussion by ThorAlex87** [UtilitechAS/amsreader-firmware#623] [Q&A] [answered]
Hi! So I just had solar installed, and when i added export into home assistant i ran into some issues with the sensors. "hour produced" is way off, by a factor of 4-5! I'm not sure if this is a home assistant issue or amshan issue, anyone have any ideas on where to start looking? Can't really say about day or month yet, but they look reasonable so far.

![Untitled](https://github.com/UtilitechAS/amsreader-firmware/assets/111507745/d8c88ba2-ed08-44a3-b4a7-add58908ae21)
![Untitled2](https://github.com/UtilitechAS/amsreader-firmware/assets/111507745/9700f105-31a9-4f68-aa7f-5e7ca66607e7)
(Raw data from the sensor is the same, so it's not an energy dashboard issue, I's just a more similar layout for comparison.)

Edit: looking at this mornings data the hourly import also looks to be way off when there is export in the same hour, but stays somewhat resonable when there is no export. I also checked the data against my grid providers data and the amshan web interface data is correct, as is the numbers shown by the energy dashboard when using the accumulated sensors. So it seems to be the hourly sensors making weird results when there is power being exported.

**Comment by ArnieO** [comment, marked as answer]
There is a known error here, a fix is coming as soon as @gskjold finds time.

**Reply by ThorAlex87** [reply]
Duh, should have checked the issues before posting...

Thanks, glad to hear it's on the list!
